cli/cmd/install.go

```diff
@@ -80,23 +80,21 @@ This command will install k8s components that will auto-instrument your applicat
 		// Check if the cluster meets the minimum requirements
 		kc := cmd.Flag("kubeconfig").Value.String()
 		details, err := autodetect.DetectK8SClusterDetails(ctx, kc, client)
-		if err == nil {
-			autodetect.CurrentKubernetesVersion = autodetect.KubernetesVersion{
-				Kind:    details.Kind,
-				Version: details.K8SVersion,
-			}
+		if !errors.Is(err, autodetect.ErrCannotDetectClusterKind) {
+			autodetect.CurrentKubernetesVersion.Kind = details.Kind
+		} else {
+			fmt.Println("Unknown Kubernetes cluster detected, proceeding with installation")
+		}
+
+		if !errors.Is(err, autodetect.ErrCannotDetectK8sVersion) {
+			autodetect.CurrentKubernetesVersion.Version = details.K8SVersion
 			if details.K8SVersion.LessThan(minK8SVersionForInstallation) {
 				fmt.Printf("\033[31mERROR\033[0m Odigos requires Kubernetes version %s or higher but found %s, aborting\n", minK8SVersionForInstallation.String(), details.K8SVersion.String())
 				os.Exit(1)
 			}
 			fmt.Printf("Detected cluster: %s Kubernetes version: %s\n", details.Kind, details.K8SVersion.String())
```

Comment:
If we failed to detect the cluster kind, can this print some confusing message?

Reply:
fixed